app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] You have declared the foreign key in the user modal so here is not necessary to delete it. Please remove this line to be DRY.